src/dss/manifest_templates/notebook_deployment.yaml.j2

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ notebook_name }}
+  namespace: {{ namespace }}
+  labels:
+    app.kubernetes.io/name: {{ notebook_name }}
+    app.kubernetes.io/part-of: dss
+    canonical.com/dss-notebook: {{ notebook_name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/part-of: dss
+      canonical.com/dss-notebook: {{ notebook_name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/part-of: dss
+        canonical.com/dss-notebook: {{ notebook_name }}
+    spec:
+      containers:
+        - env:
```

Comment:
Comming back to the GPU options. When user speciffies `--gpu nvidia` in this manifest we will add resource section as shown in [this](https://github.com/canonical/data-science-stack/blob/main/poc/nvidia/manifests/notebook-nvidia.yaml#L45) example. Create notebook should also check if the gpu is enabled on the cluster by checking the node labels. If nvidia is enabled there will be dedicated label on the node and the resource will be present on the node. If you want to check the labels and resources on node you can deploy the nvidia poc from [here](https://github.com/canonical/data-science-stack/blob/main/poc/nvidia/deploy.sh). But you need a machine with nvidia card e.g Ec2 instance [list](https://docs.aws.amazon.com/dlami/latest/devguide/gpu.html).

Reply:
to be addressed in #39 
I'll add a link to this comment there